src/main/app/src/app/zaken/zaak-koppelen-dialog/zaak-koppelen-dialog.component.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {ZakenService} from '../zaken.service';
+import {MaterialFormBuilderService} from '../../shared/material-form-builder/material-form-builder.service';
+import {AbstractFormField} from '../../shared/material-form-builder/model/abstract-form-field';
+import {Zaak} from '../model/zaak';
+import {ZaakKoppelGegevens} from '../model/zaak-koppel-gegevens';
+import {SelectFormFieldBuilder} from '../../shared/material-form-builder/form-components/select/select-form-field-builder';
+import {Validators} from '@angular/forms';
+import {UtilService} from '../../core/service/util.service';
+import {GerelateerdeZaak} from '../model/gerelateerde-zaak';
+
+@Component({
+    templateUrl: 'zaak-koppelen-dialog.component.html'
+})
+export class ZaakKoppelenDialogComponent implements OnInit {
+
+    koppelSelectFormField: AbstractFormField;
+    bronZaak: Zaak;
```

Comment:
is doelZaak niet een betere benaming? In het engels zou je zeggen 'source' is wat je gaat koppelen aan een 'target' Dus bronzaak zou dan de te koppelen zaak zijn